cosmos/operators/local.py

```diff
@@ -742,3 +742,15 @@ def __init__(self, **kwargs: str) -> None:
         raise DeprecationWarning(
             "The DbtDepsOperator has been deprecated. " "Please use the `install_deps` flag in dbt_args instead."
         )
+
+
+class DbtSourceLocalOperator(DbtLocalBaseOperator):
```

Comment:
@tatiana 

Since https://github.com/astronomer/astronomer-cosmos/pull/655 also creates a pretty similar Operator, I can wait until that is merged to follow the same naming pattern.

I tried using the DbtSourceLocalOperator name for [this](https://github.com/astronomer/astronomer-cosmos/blob/9cd46d2353dff874cc94c66baa0b146255ec42f5/cosmos/airflow/graph.py#L26) function to work.